src/raven.js

```diff
@@ -1162,24 +1162,16 @@ Raven.prototype = {
         });
     },
 
-    _makeImageRequest: function(opts) {
-        // Tack on sentry_data to auth options, which get urlencoded
-        opts.auth.sentry_data = JSON.stringify(opts.data);
-
-        var img = this._newImage(),
-            src = opts.url + '?' + urlencode(opts.auth),
-            crossOrigin = opts.options.crossOrigin;
+    _makeRequest: function(opts) {
+        var request = new XMLHttpRequest();
 
-        if (crossOrigin || crossOrigin === '') {
-            img.crossOrigin = crossOrigin;
-        }
-        img.onload = opts.onSuccess;
-        img.onerror = img.onabort = opts.onError;
-        img.src = src;
-    },
+        // if browser doesn't support CORS (e.g. IE7), we are
+        // out of luck ... report to console?
```

Comment:
Unlikely worth it since this would happen to the user, not the person who actually cares about raven-js.